<launch>
  <node name="entres_server" pkg="entres" cwd="node" type="entres_server.py" args="" output="screen" />
</launch>
